src/systemctl/systemctl.c

```diff
@@ -987,6 +987,7 @@ static int parse_argv(int argc, char *argv[]) {
         return systemctl_parse_argv(argc, argv);
 }
 
+#ifndef FUZZ_SYSTEMCTL_PARSE_ARGV
```

Comment:
I wonder, would it work and would it look better if systemctl_dispatch_parse_argv&friends moved to a new systemctl-parse.c source file and only that was included in the fuzzer binary compilation?

Reply:
I tried that approach first. But it became very messy because almost all the code needed to be moved, and all the `arg_*` variables. I didn't like the result.

Reply:
I see, that makes sense, thanks